ivy/functional/ivy/experimental/manipulation.py

```diff
@@ -1613,7 +1613,87 @@ def expand(
     ret
         Output Array
     """
-    return ivy.current_backend(x).expand(x, shape, out=out, copy=copy)
+    return ivy.current_backend(x).expand(x, shape, out=out)
```

Comment:
You shouldn't change the behavior of other functions. You can just pass `copy=None` when calling `expand`  from `put_along_axis`

Reply:
Okay, I apologise that was by mistake while solving merge conflicts.